### DISCUSSION TITLE
Different Architectures?

### DISCUSSION THREAD

**Discussion by ebgoldstein** [Doodleverse/segmentation_gym#123] [Ideas]
add a new architecture, the attention U-net.

Paper: https://arxiv.org/abs/1804.03999
PwC (for TF implementations): https://paperswithcode.com/paper/attention-u-net-learning-where-to-look-for

**Comment by ddbuscombe** [comment]
Another option is to explore use of a ResNet18 based Pyramid Attention Network (PAN) architecture, e.g. https://www.sciencedirect.com/science/article/pii/S009830042200245X

**Comment by ddbuscombe** [comment]
use https://github.com/yingkaisha/keras-unet-collection

**Comment by ebgoldstein** [comment]
With the addition of segformer - with considering how well Segformer works - i think other new architectures are on the backburner.. switching this to a discussion for now
